product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Create an EDB account for your free trial"
-description: "Creating an EDB account and signing into the BigAnimal free trial portal"
+title: "Create an EDB account to use your free credits and BigAnimal"
+description: "Creating an EDB account and signing into BigAnimal"
 navTitle: "Create an account" 
 ---
 
-You need to register with EDB to access the BigAnimal free trial and EDB's training and software downloads.
+You need to register with EDB to access BigAnimal and your $300 in free credits, as well as EDB's training and software downloads. EDB provides a free Postgres training, found here (https://www.enterprisedb.com/training).
```

Comment:
Did you mean to say "a free Postgres training"? Or just "free Postgres training" or "a free Postgres course" or some such thing?

Reply:
done